api/scanner/scanner_user.go

```diff
@@ -39,6 +55,20 @@ func getPhotoviewIgnore(ignorePath string) ([]string , error){
    	return photoviewIgnore, scanner.Err()
 }
 
+func getIgnoreFiles(ignoreAll []string) (string) {
```

Comment:
I'm not quite sure why this returns a single string rather than an array?
Nevertheless the function could maybe also be deleted if `sabhiram/go-gitignore` was used for files as well.

Reply:
Because I encountered a problem I didn't knew how to solve otherwise :) (https://stackoverflow.com/questions/52244001/go-gorm-mysql-unsupported-type-string-a-slice-of-string). But this will not be necessary if using either AlbumScannerCache or sabhiram/go-gitignore.
Disclaimer: This is the first time programming in Go. For a while I wanted to learn a new language, I strongly considered Rust, but then I stumbled on your project here, and decided to give Go a shot. So I "apologies" for the unusual Go code I might write.

Reply:
Okay it makes sense if you'd save it to the database.

No issue, your code looks clean :)